### DISCUSSION TITLE
Utilisation sans thermostat

### DISCUSSION THREAD

**Discussion by skarab22** [jmcollin78/versatile_thermostat#365] [Q&A]
Bonjour, 

Merci pour le dev qui à l'air très complet. Je contrôle actuellement mes radiateur à l'aide d'esp branché sur le fil pilote qui me permet de simuler les 4 ordre off, HG, eco et confort. Chaque radiateur est équipé d'un thermostat intégré. Il ne me parait donc pas intéressant de passer par une sonde de température supplémentaire.

Penses tu qu'il soit possible d'utiliser ton add on sans une sonde de température, afin de bénéficier de la planification et de l'ouverture des fenêtre ? 

Merci d'avance

**Comment by jmcollin78** [comment]
Bonjour,

Non on ne peut pas. C'est un pre-requis fort. Et justement le but c'est de ne surtout pas utiliser les sondes internes qui sont très fausses car trop près du corps de chauffe, pas à la place où sont les personnes, ...

Si tu as une sonde de température dans ton thermostat intégré et que tu peux la récupérer dans une sonde virtuelle, fait-le mais le résultat ne sera pas bon. La température va monter très fort tout de suite après l'allumage alors que le fond de la pièce sera froid. Autant laisser le radiateur faire tout seul du coup.

Ca coute pas cher un thermometre connecté. C'est à toi de voir.
